cse/CommentSpider.py

```diff
@@ -1,19 +1,44 @@
 import scrapy
 from scrapy.crawler import CrawlerProcess
 from scrapy.selector import Selector
-from cse.WpApiAdapter import WpApiAdapter
 import re
 import json
 from pprint import pprint
-from cse.WpApiParser import WpApiParser
-import csv
-import os
+import hashlib
+
+from cse.WpApiDataPipelineBootstrap import WpApiDataPipelineBootstrap as PipelineBootstrap
+from cse.CSVWriter import CSVWriter
 
 class CommentSpider(scrapy.Spider):
     # this spider scrapes a single article within the domain washingtonpost.com (https://www.washingtonpost.com/)
     name = 'washingtonpost.com'
     urls = [['https://www.washingtonpost.com/politics/courts_law/supreme-court-to-consider-major-digital-privacy-case-on-microsoft-email-storage/2017/10/16/b1e74936-b278-11e7-be94-fabb0f1e9ffb_story.html']]
 
+    __pbs = None
+
+
+    def __init__(self):
+        self.__pbs = PipelineBootstrap()
+        self.__pbs.setupPipeline()
+
+
+    def __setupFileWriter(self, url):
+        m = hashlib.sha256()
+        m.update(url.encode('utf-8'))
+        filename = m.hexdigest()
```

Comment:
TODO: this is rather a task of the CSVWriter than the Spider!

even better: move this into `Util.py`

Reply:
created issue: #18